packages/assets-controllers/src/TokenListController.ts

```diff
@@ -38,11 +38,11 @@ export type TokenListToken = {
 
 export type TokenListMap = Record<string, TokenListToken>;
 
-type DataCache = {
+export type DataCache = {
   timestamp: number;
   data: TokenListMap;
 };
-type TokensChainsCache = {
+export type TokensChainsCache = {
   [chainId: Hex]: DataCache;
 };
```

Comment:
If the intent is to make these package-level exports, then these need to be added to the barrel file. We should also note these in the pr description changelog as ADDED entries.

Reply:
I've realized that i no longer need the `export type DataCache`  so i removed that;
Only `TokensChainsCache` needs to be exported and its only used in `TokenDetectionController`